open-api/rest-catalog-open-api.py

```diff
@@ -399,6 +406,16 @@ class EnableRowLineageUpdate(BaseUpdate):
     action: str = Field('enable-row-lineage', const=True)
 
 
+class AddEncryptionKey(BaseUpdate):
+    action: str = Field('add-encryption-key', const=True)
+    encryption_key: Optional[EncryptedKey] = Field(None, alias='encryption-key')
```

Comment:
Shouldn't `encryption_key` be required (not an optional?)

Reply:
Good catch. I updated the spec to make this required.